mmv1/third_party/terraform/tests/resource_dataproc_cluster_test.go.erb

```diff
@@ -1135,6 +1138,11 @@ resource "google_dataproc_cluster" "basic" {
     gce_cluster_config {
       subnetwork       = google_compute_subnetwork.dataproc_subnetwork.name
       internal_ip_only = true
+      shielded_instance_config{
```

Comment:
Can you add another test that turns some of these from true to false? We have had bugs in the past where setting a boolean to false failed to actually update it.

Maybe add that as another step to this test

Reply:
Oh, these are all forcenew, nevermind.